api/scpca_portal/batch.py

```diff
@@ -62,3 +65,26 @@ def terminate_job(job) -> bool:
         batch_job_id=job.batch_job_id,
     )
     return True
+
+
+def get_jobs(batch_job_ids: List[str]) -> List[Dict] | None:
```

Comment:
Can we update this method to take in an `Iterable` of `Job` instances?

Reply:
I've updated the type hint to `Iterable["Job"]` (using a string reference to avoid a circular import with the `Job` model). But, this results in a Flake8 error (`undefined name 'Job'`). To resolve this, I've used Flake8's [In-line error ignoring](https://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html) (i.e., # noqa: F821) for this line temporarily. Let me know your thoughts on this. Thank you!